static/styles/settings.css

```diff
@@ -992,8 +996,8 @@ input[type="checkbox"] {
 
         .password_visibility_toggle {
             position: absolute;
-            left: 190px;
-            top: 28px;
+            left: 194px;
+            top: 32px;
             width: 14px;
             opacity: 0.6;
```

Comment:
Can we try rewriting the CSS for this to use `float: right` or something and not be so sensitive to precise sizing details?

Reply:
I will look into it as a follow-up to this